collections/colltest/store.go

```diff
@@ -7,19 +7,30 @@ import (
 	db "github.com/cosmos/cosmos-db"
 )
 
+type contextStoreKey struct{}
+
 // MockStore returns a mock store.KVStoreService and a mock context.Context.
-// They can be used to test collections.
-func MockStore() (store.KVStoreService, context.Context) {
+// They can be used to test collections. The StoreService.NewStoreContext
+// can be used to instantiate a new empty KVStore.
+func MockStore() (*StoreService, context.Context) {
```

Comment:
changes here allow the `MockStore` to properly match the design of the `KVStoreService`.